js/main.js

```diff
@@ -0,0 +1,46 @@
+$( document ).ready(function() {
+  // get user input && convert to float
+  var $form = $("#entry");
+  var $entries = $("#entries");
+  var total = 0;
+  var $tbody = $('tbody');
+  var input;
+
+  $form.on("submit", cashRegisterIt);
+
+  function cashRegisterIt(e) {
+    e.preventDefault();
+    var $amount = $("#newEntry").val();
+    var $amountToInteger = parseFloat($amount);
+    total += $amountToInteger;
+    $entries.append("<tr><td></td><td> $<span class='entry'>" + ($amountToInteger.toFixed(2)) + "</span><a href='#'>x</a></td></tr>");
+    var $totalText = $("#total");
+    $totalText.html("$" + (total.toFixed(2)));
+
+    // delete entries
+    var $a = $("a");
+    $a.on("click", function(e) {
+      e.preventDefault();
+      $(this).closest('tr').remove();
+    });
+
+    // editing previous entries
+      // can edit but unable to make clickaway work
+    $tbody.on('click','td',function() {
```

Comment:
i think you should add the event listener to the TR, that way `$(this)` represents each row. Which might be easier to do traversal on and subsequent removal or updates.